.azuredevops/publish-helm-library.yaml

```diff
@@ -0,0 +1,79 @@
+name: $(BuildID)
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - 'main'
+  paths:
+    include:
+      - .azuredevops/*
+      - adp-helm-library/*
+      - scripts/*
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - .azuredevops/*
```

Comment:
same here. * ?

Reply:
@ppotturi : I have not included files outside above mentioned folders.